bindings/node/package.json

```
@@ -4,6 +4,14 @@
   "description": "Official client encryption module for the MongoDB Node.js driver",
   "main": "index.js",
   "types": "index.d.ts",
+  "files": [
+    "index.js",
+    "README.md",
+    "CHANGELOG.md",
+    "lib",
```

```suggestion
    "lib",
    "src",
```

for the few who do actually build from source here

---

Ah, good point!